build.proj

```diff
@@ -26,7 +26,7 @@
     <RemoveDir Directories="$(BaseOutputPath)" />
   </Target>
 
-  <Import Project="$(ProjectDir)dependencies.targets" />
+  <Import Project="$(ProjectDir)dependencies.targets" Condition="Exists('$(ProjectDir)dependencies.targets')" />
```

Comment:
I'm not sure I understand why you added the condition.  You are adding the files, so they will always exists, right?

Reply:
Kinda described in the commit message:

> There's no need for the BfS tarball to include these files: it doesn't need to run repo-global auto-updates like submodule update.

The project files that get into the tarball are manually listed here:

https://github.com/dotnet/source-build/blob/e2ecb4c403f4d6e30137cf181ce2b43f129618b6/build-source-tarball.sh#L41-L49

I figure things that aren't used in the tarball build aren't included, and there's no need to add these two files to that list.

@crummel is that the intention? Does it need to be so explicit?

Reply:
I think we want to keep the tarball as minimal as possible so if the workaround isn't causing too much extra work I think we'd prefer to keep it out.  If it's causing issues we can include the new files.

Reply:
The only concern I have is that someone could add something to dependencies.props/targets and expect it to be set during all source-build builds. Not sure how easy that would be to debug depending on what's added.

Keeping it out of the tarball is a bit of complexity that I'm not sure we need. But maintaining that list of files is also annoying... would `*.proj`, `*.props`, `*.targets` at the root be ok?

Reply:
That sounds fine to me.